src/main/resources/AntwoordenModel_v2_2_3.csv

```diff
@@ -1,10 +1,10 @@
 v0101,v0102,v0103,v0103A,v0103B,v0104,v0201,v02A01,v02A02,v02A03,v02B01,v0301,v0302,v04A02,V04A03,V04B01,Exception,route,soortGezag,gezag_ouder1,gezag_ouder2,gezag_niet_ouder1,gezag_niet_ouder2,Index,Uitleg
-,,,,,,,,,,,,,,,,,0,N,Nee,Nee,Nee,Nee,0,Er is iets fout gegaan. Gezag is niet te bepalen.
+,,,,,,,,,,,,,,,,,0,N,Nee,Nee,Nee,Nee,0,Gezag is niet te bepalen. Neem contact op met RvIG via info@RvIG onder vermelding van BRP API gezag met de volgende foutcode: %s
 Nee,,,,,,,,,,,,,,,,,1,N,Nee,Nee,Nee,Nee,150,gezag is niet te bepalen omdat minderjarige niet in Nederland woont.
 Ja,,,,,,,,,,,,,,,,,2e,N,Nee,Nee,Nee,Nee,200,gezag is niet te bepalen omdat de geboortedatum van minderjarige niet bekend is.
 Ja,Nee_meerderjarig,,,,,,,,,,,,,,,,2m,NVT,Nee,Nee,Nee,Nee,210,Ingezeten - meerderjarig - soort gezag is NVT - gezag is niet van toepassing
 Ja,Nee_overleden,,,,,,,,,,,,,,,,2o,NVT,Nee,Nee,Nee,Nee,220,Ingezeten - minderjarig overleden - soort gezag is NVT - gezag is niet van toepassing
-Ja,Ja,,,,,,,,,,,,,,,,3e,N,Nee,Nee,Nee,Nee,300,gezag is niet te bepalen. Neem contact op met RvIG via info@RvIG onder vermelding van BRP API route 3e. Vermeld de volgende foutcode: %s
+Ja,Ja,,,,,,,,,,,,,,,,3e,N,Nee,Nee,Nee,Nee,300,gzag is niet te bepalen omdat niet kan worden vastgesteld of de persoon naar het buitenland geemigreerd geweest is
```

Comment:
Alle onverwachte fouten in de rules engine zouden moeten leiden tot route 0 met bijbehorende foutcode. Is route 3e nog nodig? Welke fout scenario leidt tot route 3e in plaats van route 0?

Reply:
Besproken tijdens een spar sessie. "e"-routes laten we voorlopig staan en besluiten over deze routes worden uitgesteld. Het afhandelen van deze routes zal worden meegenomen in een toekomstige issue waarbij foutafhandeling in de _rules engine_ in zijn algemeenheid wordt besproken.